invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/access-form.html

```diff
@@ -4,26 +4,31 @@
 
 {% from "invenio_accounts/_macros.html" import render_field, form_errors %}
 
-<div class="container">
-
+<div class="container rel-mt-2">
   {%- if current_user.is_anonymous %}
     {% set email = None %}
     {% set full_name = None %}
-    <div class="ui segment">
-      <i aria-hidden="true" class="user secret icon"></i>You are currently
-      <strong>not logged in</strong>. Do you have an account?
+    <p class="rel-mb-2">
```

Comment:
Comment: I kind of liked the segment because it made it vey visible that the user was not logged in and encourages them to use their accounts if they have one, while now it's a bit more discrete and easier to skip,

Reply:
Discussed with Lars: The segment looked too much like an input field and made the overall layout too messy. However, we could try to find a different way to make it more visible.

Not sure this is a priority atm, maybe we can do it as a separate task and this can go in as is for now?

Reply:
Sure, PR LTM! I just left a comment/personal preference, feel free to ignore it as well :)